easybuild/easyconfigs/i/iomkl/iomkl-2016.09-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2016.09-GCC-4.9.3-2.25'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description = """Intel Cluster Toolchain Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MKL & OpenMPI."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compver = '2016.3.210-GCC-4.9.3-2.25'
+
+dependencies = [
+    ('icc', compver),
+    ('ifort', compver),
+    ('OpenMPI', '1.10.2', '', ('iccifort', compver)),
```

Comment:
@edmondac the existing `iomkl/2016.07` uses OpenMPI 1.10.**3**, any reason you're not using .3 here?

Also, I see no reason to go with `2016.09` here?
Especially not since you're downgrading OpenMPI compared to `iomk/2016.07` ;)

The toolchain version is typically dictated by the release date of the most recent component, not so much the date when the toolchain was composed.

If you bump this up to OpenMPI 1.10.4, then `2016.09` *does* make sense though, since OpenMPI 1.10.4 was released Sept 1st 2016.

At first, I wasn't a big fan of retaining the `-GCC-4.9.3-2.25` suffix in the toolchain version because that makes it quite long, but since `iomkl/2016.07` is using GCC 5.x, it's probably best to make the difference clear...